### DISCUSSION TITLE
issue: about nucleation patch

### DISCUSSION THREAD

**Discussion by FURUNZ** [SeisSol/SeisSol#1021] [General]
Hi,
I have some confusion, Taking tpv5 as an example, I found that both fault and nucleation patch are set in the same plane. Does nucleation patch have any impact if not set? I saw examples without nucleation set in the.geo file of other examples. If I set shear stress exceeding the maximum static friction force in the comfort zone without setting nucleation patch, will friction rupture occur? What role does the setting of nucleation patch play

![屏幕截图 2024-01-26 095842](https://github.com/SeisSol/SeisSol/assets/92135909/7b5ca25d-4c8f-422a-856a-9b2e24bee784)

**Comment by Thomas-Ulrich** [comment]
meshing the nucleation patch may help get more accurate benchmark results (because the traction discontinuity is not smoothed out then).
In tpv5, the nucleation stress is included in the initial stress, but seissol still requires to define the nucleation stress, which is set to zero.
